specviz/core/items.py

```diff
@@ -98,10 +105,18 @@ class PlotDataItem(pg.PlotDataItem):
     """
     A PyQtGraph `~pyqtgraph.PlotDataItem` object that wraps a `DataItem` object
     (which in turn wraps a :class:`~specutils.Spectrum1D` object).
+
+    Notes
+    -----
+        Currently, the ``PlotDataItem`` class does not take in a reference to
+        the data model. This means that it is not listening to ``dataChanged``
+        signals raised from the data items within the model. Thus, if the
+        data item for this plot data item is changed, users must call the
+        ``set_data`` method on this class to re-render the plot.
```

Comment:
Minor: indentation seems wrong here, the whole block should be 4 spaces to the left.

Reply:
The body of `Notes` in docstrings are supposed to be indented (according to the [numpy docstring standard](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html), at least).